guidelines/templates/resources/release-notes.md

```diff
@@ -0,0 +1,53 @@
+> This template is dedicated to technical writers. Use it to write release notes for Kyma releases. Add them as a blog post under [`website/src/blog-posts/`](https://github.com/kyma-project/website/tree/master/src/blog-posts). Place any related screenshots under the [`assets`](https://github.com/kyma-project/website/tree/master/src/blog-posts/assets) folder. Follow the content-related guidelines and tips for writing [release notes](../../release-notes.md).
+
+<!-- Fill in the required metadata for the blog post to render properly on the "kyma-project.io" website. Remember to remove the code block. -->
+
+```
+---
+path: "/blog/{link}"
+date: "{YYY-MM-DD}"
+author: "{Name and surname}, {Role} @Kyma"
+tags:
+  - release-notes
+title: "{Release notes title}"
+---
+```
+
+<!-- This line adds a button that allows you to download the latest release. Provide the path to the release on GitHub in place of the {path} placeholder and put "Download {version number}" in place of the {content} placeholder. Remember to remove the code block. -->
+
+```
+<a class=“btn-blog” href=“{path}“><span>{content}</span></a>
```

Comment:
Please change `“` to `"` (those are a different chars). Very simple difference, but without this, button won't work :)

Final version:
`<a class="btn-blog" href="{path}">{content}</a>`

Reply:
And I also think, that the end of post is better place for this button.